### PR TITLE
Add Orange Pi PC Plus

### PR DESCRIPTION
**Pull request by eugulixes**
Checklist for testing:
* Remove toolset.
* Build an image for OPi PC Plus.
* Build a classic image based on Raspbian for RPi (any model).
* Build a mender-compatible image.
* Build an artefact.

### CODE REVIEW

File: pieman.sh [hunk 1]
```diff
@@ -290,10 +290,19 @@ case "${BUILD_TYPE}" in
...
+    if [[ "${DEVICE}" == "opi-pc-plus" ]]; then
+        dd if="${TOOLSET_DIR}/uboot-${UBOOT_VER}/u-boot-sunxi-with-spl.bin" of="${LOOP_DEV}" bs=1024 seek=8
+        sync
+    fi
+
     mount "${LOOP_DEV}p1" "${MOUNT_POINT}"
 
     rsync -a "${BOOT}"/ "${MOUNT_POINT}"
 
+    if [[ "${DEVICE}" == "opi-pc-plus" ]]; then
```
Comment (denismosolov) [inline pieman.sh]: I'd move this above the rsync
And use `"${BOOT}"/boot.scr` as a destination

File: README.md [hunk 2]
```diff
@@ -186,18 +186,20 @@ The built image will be located in the `build` directory. By the way, you can sp
...
-| Raspberry Pi [Model B and B+](https://www.raspberrypi.org/products/raspberry-pi-1-model-b/) |                |                       | 32bit                 |                             |                              |
-| Raspberry Pi [2 Model B](https://www.raspberrypi.org/products/raspberry-pi-2-model-b/)      | 32bit          | 32bit                 | 32bit                 | 32bit                       | 32bit                        |
-| Raspberry Pi [3 Model B](https://www.raspberrypi.org/products/raspberry-pi-3-model-b/)      | 32bit          | 32bit                 | 32bit                 |                             | 32bit, 64bit                 |
-| Raspberry Pi [Zero](https://www.raspberrypi.org/products/raspberry-pi-zero/)                |                |                       | 32bit                 |                             |                              |
+|                                                                                             | Alpine 3.9     | Devuan 1 «Jessie»     | Debian 10 «Buster»    | Raspbian 9 «Stretch»  | Ubuntu 16.04 «Xenial Xerus» | Ubuntu 18.04 «Bionic Beaver» |
```
Comment (polina-popova) [inline README.md]: Probably Debian should be before Devuan.

File: files/opi/boot.cmd [hunk 3]
```diff
@@ -0,0 +1,6 @@
+setenv bootargs earlyprintk console=tty0 console=ttyAMA0 root=/dev/mmcblk0p2 rootfstype=ext4 rw rootwait init=/bin/systemd
```
Comment (denismosolov) [inline files/opi/boot.cmd]: @eugulixes do we need earlyprintk? I suppose it was needed for debugging?

Comment (eugulixes) [reply files/opi/boot.cmd]: It shows the messages from the earliest phases of the Linux boot process. Some of the images already use the option. We have to either keep it in all the images or get rid of it from all the images.

Comment (denismosolov) [inline files/opi/boot.cmd]: I wonder if we can use `console=serial0,115200 console=tty1` for consistency

**Pull request merged**